docs/add_content.rst

```diff
@@ -75,19 +85,19 @@ Add Topics to Channel
 
 To create one or more new topics in a content channel, follow these steps.
 
-#. Click |plus-c| **ADD** button.
+#. Click |plus-b-b| :blue:`ADD` button.
 #. Select **Add Topics** option.
 
    .. figure:: img/add-topics.png
          :alt: Add Topics
 
          Add Topics
 
-#. Complete the topic information (title, description, language, thumbnail image) and metadata, and click the **SAVE** button. 
+#. Complete the topic information (title, description, language, thumbnail image) and metadata (tags). 
```

Comment:
Since "tags" is also a field, would it make sense to list something like "licensing information, tags, etc." to differentiate it?